features/Insights_Advisor_recommendations_filtering.feature

```diff
@@ -0,0 +1,657 @@
+          | Title    | Added       | Total risk |
+          | Bug12345 | 10 days ago | Important  |
+      And 2 another recommendations are available (but not detected)
+          | Title    | Added       | Total risk |
+          | Abc12345 | 10 days ago | Important  |
+          | Xyz12345 | 10 days ago | Moderate   |
+      And the user USER1 is already logged in into Hybrid Cloud Console
+     When user looks at Hybrid Cloud Console main page
+     Then menu on the left side should be displayed
+      And the left menu might contain these top level items
+          | Left menu item           | Required for this test |
+          | Overview                 | no                     |
+          | Releases                 | no                     |
+          | Downloads                | no                     |
+          | Advisor                  | yes                    |
+          | Subscriptions            | no                     |
+          | Cost Management          | no                     |
+          | Cluster Manager Feedback | no                     |
+          | Red Hat Marketplace      | no                     |
+          | Documentation            | no                     |
+     When user expand "Advisor" top level item
+     Then the menu should be expanded under "Advisor" top level item
+      And following new items should be displayed in the sub-menu on the left side
+          | Expanded menu item  | Required for this test |
+          | Clusters            | no                     |
+          | Recommendations     | yes                    |
+     When user select "Recommendations" menu item from this sub-menu
+     Then an "Advisor recommendations" page should be displayed right of the left menu bar
+      And widget with filter settings should be displayed
+      And table with several columns should be displayed
+          | Column name |
+          | Name        |
+          | Added       |
+          | Total risk  |
+          | Clusters    |
+      And that table should contain exactly 1 row
+          | Name        | Added       | Total risk | Clusters |
+          | Bug12345    | 10 days ago | Important  | 1        |
+      And one filter needs to be preselenced
+          | Filter name       | Tag         |
+          | Clusters impacted | 1 or more   |
+     When user clicks on "x" icon displayed on "1 or more" filter tag
+     Then table with advisor recommendation should contain recommendations names with 0 clusters
+          | Name        | Added       | Total risk | Clusters |
+          | Abc12345    | 10 days ago | Important  | 0        |
+          | Bug12345    | 10 days ago | Important  | 1        |
+          | Xyz12345    | 10 days ago | Moderate   | 0        |
+      And filter widget should consists of filter names pull-down menu and filter search input field
+     When user selects "Total risk" in filter names pull-down menu
```

Comment:
We can add here:
```
Then search input field becomes a pull-down menu with several options
  | Option name |
  | Critical           |
  | Important      |
  | Moderate      |
  | Low                |
```

Reply:
If you do this change, the "And user select "Moderate" from the rigth pull-down menu" would become "When user selects..."

Reply:
added